themes/custom-theme/components/CategoryList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
`CategoryList` page of one of the themes. whatever rendered here will be rendered in the main app. Same way we can create any many theme as we want and use the props passed in to build the page.